The codebase has introduced usage of the `const_new` features, but didn't
include it in the set of features for some reason.
--
diff --git a/Cargo.toml b/Cargo.toml
index 0ad4d2b..2b109d0 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -631,7 +631,7 @@ shellexpand = "2.1.0"
 shlex = "1.3.0"
 simplelog = "0.12.2"
 slotmap = "1.0.6"
-smallvec = { version = "1.6", features = ["union"] }
+smallvec = { version = "1.6", features = ["union", "const_new"] }
 smol = "2.0"
 sqlformat = "0.2"
 stacksafe = "0.1"
